src/descriptor.rs

```diff
@@ -169,7 +169,12 @@ impl Descriptor {
             if let Some(target_key) = row.target_key {
                 // we enforce edge order here to dedupe two way edges.
                 let (source, target) = edge_order(&row.public_key, &target_key);
-                let edge = EdgeNode::new(source.clone(), target.clone(), row.reason, row.carryover.unwrap_or(0));
+                let edge = EdgeNode::new(
+                    source.clone(),
+                    target.clone(),
+                    row.reason,
+                    row.carryover.unwrap_or(0),
```

Comment:
`unwrap_or_default()` not  more appropriate?

Reply:
Sure, they'll both yield 0 when the carryover is None. I just thought to be more explicit here is all. Can change if you prefer `unwrap_or_default` instead.

Reply:
nope... the 0 is definitely more explicit here